packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -428,6 +434,20 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
     return _ContextMenuLocation.left;
   }
 
+  // Constrain the size of the expanded child so that it does not go offscreen.
+  // See https://github.com/flutter/flutter/issues/122951
+  static double _computeScaleFactor(BuildContext context, Rect childRect){
+    final MediaQueryData mediaQuery = MediaQuery.of(context);
```

Comment:
Use `MediaQuery.paddingOf` instead, or this rebuilds whenever the `MediaQuery` changes.